<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tool_firmware_update</class>
 <widget class="QMainWindow" name="tool_firmware_update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>固件升级</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color : rgb(255, 255, 255);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>600</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>284</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>95</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border : none;

}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/*按钮普通态*/
QPushButton
{
    
    /*字体颜色*/ 
	color: rgb(50, 50, 51);
 	/*背景颜色*/  
    background-color:rgb(255,255,255);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>添加文件</string>
        </property>
        <property name="icon">
         <iconset resource="../../shuncom_qrc.qrc">
          <normaloff>:/png/shuncom_ico/png_ico/固件升级.png</normaloff>:/png/shuncom_ico/png_ico/固件升级.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border : none;</string>
        </property>
        <property name="placeholderText">
         <string>3.54.46.bin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>84</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>95</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border : none;

}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>文件名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{

    /*字体颜色*/ 
	color: rgb(255, 255, 255);
 	/*背景颜色*/  
    background-color: rgb(21, 91, 212);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>157</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 98, 102);</string>
       </property>
       <property name="text">
        <string>已选择设备数:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{

    /*字体颜色*/ 
	color: rgb(50, 50, 51);
 	/*背景颜色*/  
    background-color: rgb(255, 255, 255);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../shuncom_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
